go/enclave/crosschain/manager.go

```diff
@@ -0,0 +1,232 @@
+package crosschain
+
+import (
+	"bytes"
+	"math/big"
+	"strings"
+
+	gethlog "github.com/ethereum/go-ethereum/log"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/trie"
+	"github.com/obscuronet/go-obscuro/contracts/messagebuscontract/generated/MessageBus"
+	"github.com/obscuronet/go-obscuro/go/common"
+	"github.com/obscuronet/go-obscuro/go/enclave/db"
+)
+
+const (
+	ownerKeyHex = "6e384a07a01263518a09a5424c7b6bbfc3604ba7d93f47e3a455cbdd7f9f0682"
+)
+
+type Manager struct {
+	txOpts       *bind.TransactOpts
```

Comment:
not clear what this field is used for

Reply:
I was trying to get the auto generated golang contract to work from inside the enclave, this will be changed later on